### ISSUE TITLE
New Assembler Evaluation Interface

### ISSUE THREAD

**Issue by pbelmann** [bioboxes/rfc#151]
Bioboxes should provide a new interface for assembler evaluation tools like the already existing assembler and binning biobox.

This issue should be closed when there is

* A new assembler evaluation rfc.

* A validator for testing the implementation the assembler evaluation bioboxes.

* At least one example implementing the interface.

**Comment by pbelmann** [comment]
In PR #153 I added a new interface and a schema definition. 

It includes the following main changes:

* optional cache property for reusing intermediate results

* multiple input assemblies can be accepted

* muliple references can be accepted. I introduce a new type called fasta_dir. This poperty is important if you do not just have one reference genome but multiple ones like in metagenomics

* I renamed the interface from "Genome assembly reference-based benchmarking container" to "Genome assembly benchmarking container" since there are tools/metrics that don't need a reference.

I have quast https://github.com/pbelmann/bbx-quast as an example implementing this interface. 
By using the cache property it is able to reuse existing alignments which is very useful in metagenomics.

I will add the corresponding validator asap.
